scripts/deploy-federated-nginx.sh

```diff
@@ -0,0 +1,105 @@
+IP1=$(kubectl get node -o jsonpath="{.items[0].status.addresses[0].address}" --context cluster1)
+IP2=$(kubectl get node -o jsonpath="{.items[0].status.addresses[0].address}" --context cluster2)
+
+util::wait-for-condition 'nodePort ok in cluster1' \
+    "kubectl --namespace ${TEST_NS} --context cluster1 get service -o jsonpath='{.items[0].spec.ports[0].nodePort}' &> /dev/null" 30
+util::wait-for-condition 'nodePort ok in cluster2' \
+    "kubectl --namespace ${TEST_NS} --context cluster2 get service -o jsonpath='{.items[0].spec.ports[0].nodePort}' &> /dev/null" 30
+PORT1=$(kubectl get service -n ${TEST_NS} --context cluster1 -o jsonpath="{.items[0].spec.ports[0].nodePort}")
+PORT2=$(kubectl get service -n ${TEST_NS} --context cluster2 -o jsonpath="{.items[0].spec.ports[0].nodePort}")
+
+util::wait-for-condition 'nginx web server ok in cluster1' "curl ${IP1}:${PORT1} &> /dev/null" 120
+util::wait-for-condition 'nginx web server ok in cluster2' "curl ${IP2}:${PORT2} &> /dev/null" 120
+echo "cluster1: $(curl -s ${IP1}:${PORT1})"
+echo "cluster2: $(curl -s ${IP2}:${PORT2})"
+
+
+# Modify federated resources to update kubernetes resources in member clusters
+echo
+echo "Changing index.html in federated configmap."
+kubectl patch federatedconfigmap web-file -n ${TEST_NS} --type=merge -p '{"spec": {"template": {"data": {"index.html": "Hi K8s Fed!"}}}}'
+
+echo
+echo "Querying web server of deployed nginx application from member clusters."
+util::wait-for-condition 'web content changed in cluster1' "(curl ${IP1}:${PORT1} | grep '^Hi K8s') &> /dev/null" 120
+util::wait-for-condition 'web content changed in cluster2' "(curl ${IP2}:${PORT2} | grep '^Hi K8s') &> /dev/null" 120
+echo "cluster1: $(curl -s ${IP1}:${PORT1})"
+echo "cluster2: $(curl -s ${IP2}:${PORT2})"
+
+echo
+echo "Updating override of federated deployment nginx to increase 'replicas' to 2 in cluster2."
+kubectl patch federateddeployment nginx -n ${TEST_NS} --type=merge --patch \
+    '{"spec" : {"overrides": [{"clusterName" : "cluster2", "clusterOverrides": [{"path": "spec.replicas", "value" : 2}]}]}}'
+echo "Checking 'replicas' of deployment nginx in cluster2."
+util::wait-for-condition 'replicas updated in cluster2' \
+    "(kubectl get deployment nginx -n ${TEST_NS} --context cluster2 -o jsonpath='{.status.replicas}' | grep "^2$") &> /dev/null" 120
+
+echo
+echo "Updating placement to include only 'cluster2' so that the deployment will be removed from 'cluster1'"
+kubectl patch federateddeployment nginx -n ${TEST_NS} --type=merge --patch '{"spec": {"placement": {"clusters": [{"name": "cluster2"}]}}}'
+echo "Checking status of federated deployment nginx."
+util::wait-for-condition 'federated deployment status updated' \
+    "(kubectl get federateddeployment nginx -n ${TEST_NS} -o jsonpath='{.status.clusters[*].name}' | grep "^cluster2$") &> /dev/null" 120
+
+echo
+echo "Federation nginx application is done."
```

Comment:
`s/Federation nginx application is done/Successfully completed KubeFed walkthrough/`